src/main/scala/redis/Sentinel.scala

```diff
@@ -190,7 +190,7 @@ abstract class SentinelMonitored(system: ActorSystem, redisDispatcher: RedisDisp
   def withSlavesAddr[T](initFunction: Seq[(String, Int)] => T): T = {
     import scala.concurrent.duration._
 
-    val fslaves = Future.sequence(sentinelClients.values.map(_.slaves(master)))
+    val fslaves = Future.sequence(sentinelClients.map({ case (_, client) => client.slaves(master) }))
```

Comment:
Using `.values` here broke `SentinelMonitoredRedisClientMasterSlavesSpec` with Scala 2.13. I didn't look into the exact reason but using `.values` was discouraged even before Scala 2.13 so just to be on the safe side I replaced all of them with `.map`.